scripts/run-benchmarks.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+const { spawn, execSync } = require('@lerna/child-process')
+const glob = require('glob')
+const { join } = require('path')
+const { readFile, writeFile } = require('fs')
+const { promisify } = require('util')
+
+const pReadFile = promisify(readFile)
+const pWriteFile = promisify(writeFile)
+const PKG_DIR = join(__dirname, '../')
+
+async function getAllBenchmarkResults() {
+  const results = []
+  const files = glob.sync(`${PKG_DIR}/packages/**/reports/*-benchmarks.json`)
```

Comment:
Is there anyway we can avoid writing the benchmarks to disk and then reading them from disk again? 
Having the result in the disk is useful for manual analysis but maybe in this case we can just get the results in memory.

Reply:
One of the reason why I didn't tweak this is because for the other PR where we are running the benchmarks for rum #302 , We have to again store the results based on some other format and we cannot keep storing them in memory and run the final analysis later on since we are depending on each specific runs on the packages/*. 

We could easily do this if it's only for a single package itself. I wanted to keep this extensible. We can come up with some ideas once everything is in place.